src/utils/fs.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import fs from 'fs';
+import * as tar from 'tar';
+
+export const extractTarBall = async (
+	filePath: string,
+	directoryPath: string,
```

Comment:
would it be more readable if we name it to something representing one is the tar file path and the other one is the destination directory / extraction directory? Ex: `tarFilePath/srcFilePath`, `extractionDirPath/destDirPath`